web_tree_dynamic_colored_field/__openerp__.py

```diff
@@ -1,37 +1,22 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Damien Crier
-#    Copyright 2015 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2015 Camptocamp SA, Damien Crier
```

Comment:
copyrights in files apply to the author of the file, and we didn't change anything meaningful there